deployments/ecmps/release-v2.0-fix/Sprint12/6403/defaults_master_data_relationships_load.sql

```diff
@@ -0,0 +1,642 @@
+-- Clear invalid entries for AKSF and SORX 
+update camdecmpsmd.cross_check_catalog_value 
+set value2=null
+WHERE cross_chk_catalog_value_id=(select cross_chk_catalog_value_id
+									from camdecmpsmd.cross_check_catalog_value 
+									where cross_chk_catalog_id=(select cross_chk_catalog_id 
+																from camdecmpsmd.cross_check_catalog
+																where cross_chk_catalog_name='Parameter to Category') 
+																	and value1='AKSF' 
+																	and value2='DEFAULT');
+
+-- Update all the existing "Parameter to Category" "SORX" Value 2 to "DEFAULT".
```

Comment:
**Just a note**, and I realize you used previous SQL as the model, but using "camdecmpsmd.cross_check_catalog_value" being the main table and also used in the where clause is very odd.  **In the future** I would use something like the following.

```
update  camdecmpsmd.cross_check_catalog_value 
   set  value2='DEFAULT'
 where  cross_chk_catalog_id = (
                                select  cross_chk_catalog_id
                                  from  camdecmpsmd.cross_check_catalog
                                 where  cross_chk_catalog_name = 'Parameter to Category')
                                   and  value1='SORX'
                               )
   and  value1 = 'SORX';
```

Reply:
Thanks for the reminder Dwayne!